src/System.Resources.Writer/src/System/Runtime/Versioning/MultitargetingHelpers.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Diagnostics.Contracts;
+namespace System.Runtime.Versioning
+{
+ 
+    internal static class MultitargetingHelpers
+    {
+        // default type converter
+        private static Func<Type, String> defaultConverter = (t) => t.AssemblyQualifiedName;
```

Comment:
We do not need to store pointer to t.AssemblyQualifiedName in a delegate - we should just call the method directly where it is used.